### ISSUE TITLE
Intention to identify which macros define symbols?

### ISSUE THREAD

**Issue by hlship** [cursiveclojure/cursive#242]
I have no idea how you would accomplish this but ...

![tutorial-service_service_-__tutorial_service__-_pedetal-tutorial-service_-____workspaces_annadale_webframeworks_pedestal_tutorial_pedetal-tutorial-service_](https://f.cloud.github.com/assets/52660/1923752/8de9945e-7e19-11e3-9640-dfb9b6121b55.png)

I'd like to be able to Alt-Return with the cursor on `session`.  Cursive would search the `middlewares` namespace and find this:

![io_pedestal_service_http_ring-middlewares_-__leiningen__io_pedestal_pedestal_service_0_2_2__-_pedetal-tutorial-service_-____workspaces_annadale_webframeworks_pedestal_tutorial_pedetal-tutorial-service_](https://f.cloud.github.com/assets/52660/1923774/c1157bea-7e19-11e3-8d78-5241697d8bf3.png)

Basically a form starting with `^def.`* then the symbol that wasn't previously matched.

It would then raise a dialog to ask whether `definterceptorfn` is a macro that defines a symbol. Perhaps it would identify the fully qualified name.

Cursive would then add that information persistently for the project so that anywhere else `definterceptorfn` was used, it would know that it was defining a symbol.

Thoughts?

**Comment by cursiveclojure** [comment]
Closing this in favour of the issue linked by @sfnelson
